<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerWorkflowSegmentationModule</class>
 <widget class="qSlicerWidget" name="qSlicerWorkflowSegmentationModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>795</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="DisplayCollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="BufferGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Buffer</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ControlsGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="MessagesGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Messages</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="qMRMLNodeComboBox" name="WorkflowSegmentationNodeComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLWorkflowSegmentationNode</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="ctkCollapsibleButton" name="WorkflowsCollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Workflows</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="ctkCollapsibleButton" name="AdvancedCollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="RefreshButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Refresh Workflow Displays</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ToolEditorGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Tool Editor</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ToolSummaryGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Tool Sumary</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerWorkflowSegmentationModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>WorkflowSegmentationNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
